### PR TITLE
Active users

### PR DESCRIPTION
**Pull request by solth**

### CODE REVIEW

File: Kitodo/src/main/webapp/pages/activeUsers.xhtml [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-    <ui:define name="pageTabView">
-        <p:tabView id="activeUsersTabView">
-            <p:tab id="activeUsersTab" title="#{msgs.userList}">
-                <h:form id="activeUsersForm">
-                    <p:poll interval="10" update="tableActiveSessions"/>
-                    <p:dataTable id="tableActiveSessions" value="#{SessionForm.activeSessions}" var="userItem">
-                        <p:column id="userName" headerText="#{msgs.user}">
-                            <h:outputText value="#{userItem.userName}"/>
-                        </p:column>
-                        <p:column id="lastRequest" headerText="#{msgs.lastAccess}">
```
Comment (Beacze) [inline Kitodo/src/main/webapp/pages/activeUsers.xhtml]: Probably this message is right now not used anymore an can be removed.

File: Kitodo/src/main/webapp/pages/activeUsers.xhtml [hunk 2]
```diff
@@ -1,45 +0,0 @@
...
-                        </p:column>
-                    </p:dataTable>
-                </h:form>
-            </p:tab>
-        </p:tabView>
-    </ui:define>
-
-    <ui:define name="breadcrumbs">
-        <li><h:link value="#{msgs.startPage}" outcome="start"/></li>
-        <li><a href="">#{msgs.activeUsers}</a></li>
```
Comment (Beacze) [inline Kitodo/src/main/webapp/pages/activeUsers.xhtml]: Probably this message is right now not used anymore an can be removed.

File: Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml [hunk 3]
```diff
@@ -55,22 +55,6 @@
...
-                            class="batch-count bc-info">#{SessionForm.activeSessions.size()}</h:panelGroup></a>
-                        <ul>
-                            <li><h3>#{msgs.activeUsers}</h3></li>
-                            <ui:repeat
-                                value="#{SessionForm.activeSessions.subList(0, (SessionForm.activeSessions.size() gt 3) ? 3 : SessionForm.activeSessions.size())}"
-                                var="activeSession">
-                                <li><div><span class="nav-item-col">#{activeSession['userName']}</span></div></li>
-                            </ui:repeat>
-                            <li><h:link outcome="activeUsers"
-                                        value="#{msgs.allActiveUsers} (#{SessionForm.activeSessions.size()})"/></li>
```
Comment (Beacze) [inline Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml]: Probably this message is right now not used anymore an can be removed.

File: Kitodo/src/main/webapp/pages/activeUsers.xhtml (hunk 2 above)
Comment (solth) [reply Kitodo/src/main/webapp/pages/activeUsers.xhtml]: The `activeUsers` message is still used in line 172 of `SearchForm.java`, so I didn't remove it.

File: Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml (hunk 3 above)
Comment (solth) [reply Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml]: Removed.

File: Kitodo/src/main/webapp/pages/activeUsers.xhtml (hunk 1 above)
Comment (solth) [reply Kitodo/src/main/webapp/pages/activeUsers.xhtml]: Removed.

**Pull request merged**